app/views/dashboard/index.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>Dashboard#index</h1>
-<p>Find me in app/views/dashboard/index.html.erb</p>
+
+<nav class="navbar navbar-dark bg-primary" style="background-color: #e3f2fd;">
```

Comment:
I think we should do navigation bar in some separate partial and just render it in application layout :) And no inline styles :D